### DISCUSSION TITLE
how to use webp ?!

### DISCUSSION THREAD

**Discussion by ppeyman** [breezefront/community#30] [1. Help] [answered]
Hi dear breezefront team 
Thank you for your amazing theme i really happy to use it 

and i have a question about in webp image who uploaded on your demo homepage.

1. how you upload it ?
2. as you know magento 2 doesn't compatible webp image on homescreen are you use any module ? 
3. could you help me which module you used for it ?

**Comment by vovayatsyuk** [comment, marked as answer]
Yes, magento doesn't allow to use webp images yet :(. 

We've manually replaced png/jpg with webp images in theme installer files: https://github.com/breezefront/theme-frontend-breeze-evolution/tree/master/etc/marketplace
